src/test/scala/gitbucket/core/GitBucketCoreModuleSpec.scala

```diff
@@ -33,6 +33,7 @@ class GitBucketCoreModuleSpec extends FunSuite {
       .withPort(3306)
       .withUser("sa", "sa")
       .withCharset(Charset.UTF8)
+      .withServerVariable("log_syslog", 0)
```

Comment:
Normal Windows users do not have permission to access the event log and the test will fail.